apps/nameguard.io/src/components/organisms/ContactUsForm.tsx

```diff
@@ -274,30 +253,19 @@ export const ContactUsForm = ({ title }: ContactUsFormProps) => {
               >
                 Telegram (optional)
               </label>
-              <div className="relative">
-                <span className="text-sm leading-5 font-medium text-gray-500 absolute left-2 top-2.5">
-                  @
-                </span>
-                <input
-                  type="text"
-                  id="telegram"
-                  autoComplete="off"
-                  disabled={isLoading}
-                  onChange={handleInputChange}
-                  name={FormFields.Telegram}
-                  className={cc([
-                    "block w-full rounded-md pr-3.5 pl-6 py-2 text-gray-900 shadow-sm border border-gray-300 placeholder:text-gray-400 focus:border-gray-600 hover:border-gray-400 outline-none sm:text-sm sm:leading-6",
-                    {
-                      "border-red-300": validationErrors[FormFields.Telegram],
-                    },
-                  ])}
-                />
-                {validationErrors[FormFields.Telegram] && (
-                  <span className="mt-2 text-sm font-normal text-red-600">
-                    {validationErrors[FormFields.Telegram]}
-                  </span>
-                )}
-              </div>
+              <Input
+                type="text"
+                id="telegram"
+                slotPosition="left"
+                slot="@"
+                autoComplete="off"
+                disabled={isLoading}
+                onChange={handleInputChange}
+                name={FormFields.Telegram}
+                error={validationErrors[FormFields.Telegram]}
+              >
+                <Input.Slot>@</Input.Slot>
+              </Input>
             </div>
             <div className="gap-1">
```

Comment:
@eduramme It seems the input styling for the form fields is not consistent. For example:

![CleanShot 2024-09-11 at 15 06 27](https://github.com/user-attachments/assets/3ef6941e-e472-4fa6-925e-4ef7a5b8ef73)

The text in blue has a different vertical alignment than the top 2 fields.

The text in red is styled differently than the other fields.

Please ensure the text styling on all these fields is always exactly the same.

I suppose part of this is that we need to update the `<textarea ...` below this comment to become a `<TextArea` as defined in our UI library. Please include the goal of updating all text areas across our monorepo as part of this PR. Otherwise we will have inconsistent UIs as we can see from the screenshot above.

Thanks

Reply:
@lightwalker-eth I've replaced `textarea` by `TeaxtArea` everywhere